app/services/description_generator.rb

```diff
@@ -88,7 +88,7 @@ def created_date
       type: 'creation',
       date: [
         {
-          value: date.respond_to?(:to_edtf) ? date.to_edtf : date.to_s,
+          value: date.respond_to?(:edtf) ? date.edtf : date.to_s,
```

Comment:
Sorbet said so.

Reply:
I'm not sure I understand.  what was wrong with `#to_edtf`?

Reply:
I don't always understand what Sorbet does, either.

Reply:
I thought we were OK with some folks using the pre-push hook automation, though?

Reply:
My understanding was that only changed .rbi files.

Reply:
:man_shrugging: I'm happy to twiddle Sorbet-related stuff in a follow-up PR, though.